backend/src/main/java/com/happy/friendogly/club/dto/response/FindClubParticipatingResponse.java

```diff
@@ -0,0 +1,42 @@
+package com.happy.friendogly.club.dto.response;
+
+import com.happy.friendogly.club.domain.Club;
+import com.happy.friendogly.club.domain.Status;
+import com.happy.friendogly.pet.domain.Gender;
+import com.happy.friendogly.pet.domain.SizeType;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Set;
+
+public record FindClubParticipatingResponse(
+        Long id,
```

Comment:
```suggestion

public record FindParticipatingClubResponse(
        Long id,
```

여기도 자연어랑 가까운 네이밍이면 좋겠어요

Reply:
자연어 vs 컨벤션 ..?

Reply:
이건 컨벤션으로 정한 내용이 없는 것 같아요! 저는 자연어에 가까운(영어 문법적으로 맞는) 걸 선호하는 편인데
크게 거슬리게 보이지는 않아서 위브가 보기에 편하거나 자연스러운 것 같으면 그대로 둬도 될 것 같아요~~